### ISSUE TITLE
Neatline error associating Omeka item to Neatline record if public theme has custom.php file

### ISSUE THREAD

**Issue by jeremyboggs** [scholarslab/Neatline#268]
Using Omeka 2.0 and 2.1, if the active public theme contains a `custom.php` file with _any_ function defined, Neatline gets an error when associating an Omeka Item to a Neatline record using the search form. The error in the Apache log:

```
[Tue Dec 03 10:42:53 2013] [notice] child pid 7036 exit signal Segmentation fault (11)
[Tue Dec 03 10:43:42 2013] [error] [client 127.0.0.1] PHP Fatal error:  Cannot redeclare here_is_a_function_yo() (previously declared in /Users/jeremy/Sites/omeka/themes/berlin/custom.php:5) in /Users/jeremy/Sites/omeka/themes/berlin/custom.php on line 5, referer: http://omeka.dev/admin/neatline/editor/2
```

Unsure what the problem actually is, but it seems the Neatline editor may be loading the public theme's custom.php twice?

**Comment by waynegraham** [comment]
verified this behavior in PHP 5.4.22 and and 5.3.6

```
[Tue Dec  3 11:16:54 2013] ::1:60147 [500]: /neatline/records/1 - Cannot redeclare foobar() (previously declared in /Applications/MAMP/htdocs/omeka-2.1.2/themes/berlin/custom.php:6) in /Applications/MAMP/htdocs/omeka-2.1.2/themes/berlin/custom.php on line 6
```

Contents of the `custom.php` file in **bernlin**, **default**, and **seasons** themes:

``` php
<?php

function foobar()
{
  echo "bar";
}
```

Could this be an issue with the `Zen_Loader`?

**Comment by davidmcclure** [comment]
Yeah, I see it too. Not sure - it's triggered by the call to `all_element_texts('item')` in the `item.php` template:

https://github.com/scholarslab/Neatline/blob/develop/views/shared/exhibits/item.php#L15

I thought it could be an issue with how NL mocks out the view in this flow to make it possible to do this step inside of background processes, but I commented out the function call that does that, and the problem persists.

@clioweb, do you see anything in the nl_getItemMarkup() helper that looks awry? (Other than the call to `nl_mountView()`, which was what I tried getting rid of.)

https://github.com/scholarslab/Neatline/blob/develop/helpers/Views.php#L19

**Comment by jeremyboggs** [comment]
I get this error in Neatline 2.0.4 as well, on Omeka 2.0, so this issue seems to have been around for a while.

**Comment by davidmcclure** [comment]
Fixed by 8eba9aa5e89f5e4c90d40d928acb6f356a6901da, by using the `render()` method on Omeka's view class instead of `partial()` to render the item template. Not totally clear about what's going on under the hood here, but this works fine.
